lib/pdk/cli/convert.rb

```diff
@@ -14,14 +14,15 @@ module PDK::CLI
     flag nil, :'default-template', _('Convert the module to use the default PDK template.')
 
     run do |opts, _args, _cmd|
-      require 'pdk/module/convert'
-      require 'pdk/cli/util'
-
-      PDK::CLI::Util.ensure_in_module!(
-        check_module_layout: true,
-        message:             _('`pdk convert` can only be run from inside a valid module directory.'),
-        log_level:           :info,
-      )
+      # Write the context information to the debug log
+      PDK.context.to_debug_log
+
+      unless PDK.context.is_a?(PDK::Context::Module)
+        raise PDK::CLI::ExitWithError.new(
+          _('`pdk convert` can only be run from inside a valid module directory.'),
+          log_level: :error,
```

Comment:
nit: `PDK::CLI::ExitWithError` defaults to the `:error` log level

Reply:
Fixed